beeline/pom.xml

```diff
@@ -105,19 +117,40 @@
       <artifactId>super-csv</artifactId>
       <version>${super-csv.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
```

Comment:
arent we already using slf4j libraries in beeline today? How has this been compiling without explicit dependencies? transitive?
also we should specify a scope for these dependencies.

Reply:
It is being used, but the dependencies have been transitive for a long time (d'oh).

No scope required. Technically don't need it for compile, but it's no harm.